pkg/asset/internal/templates.go

```diff
@@ -265,7 +265,7 @@ spec:
       hostNetwork: true
       containers:
       - name: checkpoint
-        image: quay.io/coreos/pod-checkpointer:a936ee1149af95c7696a261e739b050ff00c6288
+        image: quay.io/coreos/pod-checkpointer:99ddc731171c0ee860ca482dd25db161a5ceb304
```

Comment:
Could be done in another PR, and we usually do, however I have become less familiar with the testing of bootkube as things have changed, and just wanted to make sure the new checkpointer is used in any / all tests ran.